packages/scaffold-utils/src/SolanaConstantsUtil.ts

```diff
@@ -0,0 +1,28 @@
+import { PublicKey } from '@solana/web3.js'
+
+export const SolConstantsUtil = {
+  HASH_PREFIX: 'SPL Name Service',
+  NAME_OFFERS_ID: new PublicKey('85iDfUvr3HJyLM2zcq5BXSiDvUWfw6cSE1FfNBo8Ap29'),
```

Comment:
What are these keys? How do we create those? Should we move them to environment variables? Preferably a JSDoc for `SolConstantsUtil` would be nice with short descriptions about the keys.